test/lib/form/validators.test.ts

```diff
@@ -251,6 +249,58 @@ describe("Form Validators", () => {
     });
   });
 
+  describe("phoneNumber", () => {
+    beforeEach(() => {
+      ({ errorMessage, applies } = Validators.phoneNumber(
+        expectedErrorMessage
+      ));
+    });
+
+    it("should error for obviously invalid phone numbers", () => {
+      expect(applies("i am not a phone number")).toBe(true);
+      expect(applies("0121 D0 0NE")).toBe(true);
+      expect(applies("+44       ")).toBe(true);
+    });
+
+    it("should not error for UK mobile numbers", () => {
+      expect(applies("07712 345678")).toBe(false);
+      expect(applies("+447712 345678")).toBe(false);
+    });
+
+    it("should not error for valid satellite numbers", () => {
+      expect(applies("+881612345678")).toBe(false);
+      expect(applies("+870773166238")).toBe(false);
+      expect(applies("+881677722191")).toBe(false);
+      expect(applies("+8821663225558")).toBe(false);
+    });
+
+    it("should permit spaces", () => {
+      expect(applies("077 12 345 678")).toBe(false);
+      expect(applies("+44 77 12 345 67 8")).toBe(false);
+      expect(applies("+8 8161 234 56 78")).toBe(false);
+      expect(applies("+87 07731 66 238")).toBe(false);
+      expect(applies("+88 167 772 21     91")).toBe(false);
+      expect(applies("+ 882   1663225558")).toBe(false);
+    });
+
+    it("should permit dashes and hyphens", () => {
+      expect(applies("07712-345-678")).toBe(false);
+      expect(applies("+44-7712345-678")).toBe(false);
+      expect(applies("+8816 - 12345678")).toBe(false);
+    });
+
+    it("should permit brackets", () => {
+      expect(applies("(0)7712345678")).toBe(false);
+      expect(applies("(+44) 7712345678")).toBe(false);
+      expect(applies("(+8816)12345678")).toBe(false);
+    });
+
+    it("should permit '00' instead of '+' for country codes", () => {
+      expect(applies("00447712345678")).toBe(false);
+      expect(applies("008816-12345678")).toBe(false);
+    });
+  });
+
```

Comment:
This is the minimum amount of testing I chose to do on the validator that calls `libphonenumber`, mostly to confirm my understanding of how it works and confirm I'd configured it correctly (you need to set a default country code 🤦🏻‍♂️ ).  Might add more test cases if Linda and the team want more confidence in the types of numbers their customers tend to try and add.

Reply:
I think this is more than enough. Otherwise, we run the danger of writing tests for a library that already has tests checking the same thing. 

Main thing for me is that we're confident with what the library is doing and understand the interface and when we expect validation errors to be thrown